### ISSUE TITLE
[1.15.2]exe program, no Chinese translation

### ISSUE THREAD

**Issue by EnterFor** [superzanti/ServerSync#175]
#### Serversync Version: 
3.4.0
#### Minecraft Version: 
1.15.2
#### Issue: 
no Chinese translation..
Must I switch to the jar version?
I think the exe version of the icon looks better..

**Comment by rheimus** [comment]
The exe version is basically just a zip file with the java files inside it, it should be identical.

**Comment by EnterFor** [comment]
![image](https://user-images.githubusercontent.com/10114685/78640559-8ce54880-78e2-11ea-9405-961e430c13ee.png)
Yes, I found it, but it doesn't show Chinese when I run it.
Does this have anything to do with me changing the exe program file name?

**Comment by rheimus** [comment]
Changing the name of the file should be fine.

Does it have some translation or none at all? There are several strings that have not been hooked into the resource bundle yet.

**Comment by EnterFor** [comment]
None at all.
It only shows English
![image](https://user-images.githubusercontent.com/10114685/78641624-15b0b400-78e4-11ea-91cc-740cb1acdabf.png)
![image](https://user-images.githubusercontent.com/10114685/78641639-1c3f2b80-78e4-11ea-9675-de959cb48beb.png)
The Chinese shown in the figure is that I annotated the Mod so that I could better view it

**Comment by rheimus** [comment]
Could you send me the client logs please.

I get the feeling that the client is failing to match your locale string to the resource bundle.

**Comment by EnterFor** [comment]
[serversync-client.log](https://github.com/superzanti/ServerSync/files/4442616/serversync-client.log)
Is this it?

**Comment by rheimus** [comment]
Apologies, github decided to auto close this.

Though the latest merge should indeed fix the issue, looks like ServerSync was never loading resource bundles properly.

**Comment by rheimus** [comment]
Early test files can be found here: https://github.com/superzanti/ServerSync/actions/runs/72526484

**Comment by EnterFor** [comment]
![image](https://user-images.githubusercontent.com/10114685/78654132-b6a86a80-78f6-11ea-933f-3084617b14af.png)
Ummmmm....

**Comment by rheimus** [comment]
😢 Bummer, probably need to play with the encoding.

**Comment by EnterFor** [comment]
If there is no way, it doesn't matter~~~~~~~~

**Comment by rheimus** [comment]
The issue is that by default in java 7 & 8 resource bundles are encoded in ISO.

I should be able to work around this and get the bundle as utf-8.

**Comment by rheimus** [comment]
Give this one a try https://github.com/superzanti/ServerSync/actions/runs/72674632

I have just converted the file to use unicode escapes for now, shall work on a better solution later.

**Comment by EnterFor** [comment]
![image](https://user-images.githubusercontent.com/10114685/78669405-cc298e80-790e-11ea-8e85-d6035fc79bc3.png)
Nice!!!!!!!!!!!!!!!!!!!!!

**Comment by rheimus** [comment]
👍, just need to actually hook up the various strings in the app to the resource bundles now.

**Comment by EnterFor** [comment]
Wait a minute, I found multiple extra strings, I think I can supplement the translation

**Comment by EnterFor** [comment]
Sorry, I found a flaw in the translation. It has been resubmitted.

If there are no other questions, this question can be closed. Thanks again for your patience. I seem to be a bit wordy..xD.
